xilem/src/driver.rs

```diff
@@ -312,7 +311,10 @@ where
         let fonts = std::mem::take(&mut self.fonts);
 
         for root in state.roots() {
-            if let Some(root_widget) = root.get_layer_root(0).downcast::<DynWidget>() {
+            if let Some(root_widget) = root
+                .get_layer_root(0)
+                .downcast::<masonry::widgets::ContentHost>()
+            {
```

Comment:
I'm working on a different PR that changes this ... but this preserves the current behavior.